src/components/Settings/Settings.tsx

```diff
@@ -49,7 +51,23 @@ const Settings = () => {
 
   // Submits the form. Triggered by pressing the save button
   const handleSubmit: FormEventHandler = (e) => {
-    e.preventDefault();
+    // e.preventDefault();
+    if (activeNode) {
+      const updatedUser: User = {
+        id: deviceID,
+        longName: deviceName,
+        shortName: deviceNickname,
+        macaddr: activeNode ? activeNode.data.user!.macaddr : [],
+        hwModel: activeNode ? activeNode.data.user!.hwModel : 0,
+        isLicensed: activeNode ? activeNode.data.user!.isLicensed : true,
+      };
+
+      invoke("update_device_user", { user: updatedUser }).catch((e) => {
```

Comment:
Commenting out of curiosity -- I know this has to be a string, but is there a way to have it not hardcoded when calling the event handler from the frontend? If not, do we want a standardized way of storing strings in the frontend (e.g. some data structure)? 

It seems like another level of abstraction and unnecessary for this PR, but I'm wondering if sometime in the future it would be nice to have a file with all the handler name strings and a description of what each of them does (and we'll invoke them using that). 

@sherry1iu and @ajmcquilkin - might be unnecessary, thoughts? I don't fully understand event handlers, so feel free to shoot this down

Reply:
Oh yeah this is probably a bigger issue that warrants its own ticket. The way we typically are handling these command calls is by creating a new redux saga handler that can then abstract over the call so that the string is only in one location. Not blocking this PR on this point, but I would encourage refactoring this into a similar form to the way we send string messages ([link](https://github.com/ajmcquilkin/Meshtastic-emergency-response-client/blob/21861dacb5d654b04945b8ddf0f6236ad310a61f/src/features/device/deviceSagas.ts#L65)). Also happy to chat on this one